src/api/localResolvers/loanSearch.js

```diff
@@ -24,7 +24,7 @@ export default () => {
 				gender: '', // expects a string
 				countryIsoCode: [], // expects an array of strings
 				sectorId: [], // expects an array of ints
-				sortBy: '', // expects a string
+				sortBy: null, // expects a string
```

Comment:
With `enum` types in graphql you can pass null rather than an empty string if you intend for the default value to be used.